01_homework/hw2_task5.cpp

```diff
@@ -0,0 +1,81 @@
+#include <iostream>
+#include <chrono>
+#include <array>
+#include <vector>
+#include <thread>
+#include <numeric>
+
+constexpr int reps = 2000000;
+
+template<typename T>
+void stress_function(T a, T &b) {
+    for(int i = 0; i < reps; i++) {
+        b = a*b;
+    }
+}
+
+struct Int64 {
+    alignas(64) int value;
```

Comment:
jakbyś przerzucił alignas do struktury `struct alignas(64) Int64` to myślę, że zadziałałoby zgodnie z oczekiwaniami.

Reply:
Z nieznanego mi powodu zadna z tych wersji nie rozwiązuje problemu. Możliwe ze to jakas indywdualna zachcianka mojej maszyny. Aktualnie nie bardzo mam dostep do innej by sprawdzic.

Reply:
Spróbuj na repl.it :)